tests/__init__.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+
+smartseq2_paired_ends_query = \
```

Comment:
i still don't think this belongs here.

Reply:
where do we put reused variables, in infra.py?

Reply:
I don't know.  search_queries.py?

I just know it doesn't belong in `__init__.py`